en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+
+## Develop
+
+In this section, let's develop the application that retrieves COVID-19 related statistics.
+
+### Create a project and add a REST API component
+
+1. Sign in to the Choreo Console at https://consolev2.preview-dv.choreo.dev/.
+
+2. Expand the drop-down menu for projects and click **Create Project**.
+
+    ![Create project](../assets/img/tutorials/rest-api/create-project.png){.cInlineImage-full}
+
+3. Enter the name and the description for the project as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `COVID-19Stats`                       |
+    | **Description** | `Get statistics relating to COVID-19` |
+
+4. Click **Create**.
+
+### Add a REST API component
+
+Let's create a new REST API component as follows:
+
+1. On the **Components** page, click **Create**.
+
+2. Click on the **REST API** card.
+
+3. Click on **Create an API from scratch**.
+
+4. Enter a name and a description for the API as follows:
+
+    | **Field**       | **Value**             |
+    |-----------------|-----------------------|
+    | **Name**        | `Statistics`          |
+    | **Description** | `COVID-19 Statistics` |
+
+5. Click **Create**.
+
+The `Statistics` REST API opens on a separate page.
+
+### Design the REST API
+
+Let's design the REST API as follows:
+
+1. To open the REST API component in the Web Editor, click **Edit with VSCode Online**.
+
+    Then click **Open Link in New Tab** in the message that appears to indicate that the code server is ready.
+
+2. On the left navigator, click **service.bal**  to open the low-code diagram of your REST API component.
+
+3. Add a new construct and an API resource as follows:
+
+    1. Click on the first **+** icon to add an HTTP construct. Select the **Service** construct type,
+
+        ![Add service construct](../assets/img/tutorials/rest-api/add-service-construct.png){.cInlineImage-half}
+
+        Select HTTP as the service type, and enter the following information.
+
+        | **Field**             | **Value**             |
+        |-----------------------|-----------------------|
+        | **Service Base Path** | `COVID19-Statistics`  |
+        | **Listener Port**     | `9090`                |
+
+        Select the **Define Inline** check box, and then click **Save**.
+
+    2. Expand the construct you created by clicking on it. Then click the **EDIT** icon of the default `GET` resource within the construct.
+
+    3. Enter `stats/[string country]` in the **Path** field.
+
+    4. Click **Advanced** and in the **Return Type** field, enter `json|error` to get the return in `json` or `error1` format.
+
+    5. Click **Save API**.
+
+4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
+
+    1. Click on the API resource you edited, and then click on the first **+** icon after the **Start** statement.
```

Comment:
Shall we add a screenshot here?

Reply:
Isn't it intuitive? A screenshot per plus icon would result in too many screenshots and make the tutorial appear to lengthy at a glance.